src/components/Card/Card.Styled.ts

```diff
@@ -1,13 +1,13 @@
 import styled from "styled-components";
 
-export const Card = styled.div<{ $filmImage?: string }>`
+export const Card = styled.div<{ filmImage?: string }>`
```

Comment:
Nie wiem czy Ci to podpowiadałem, ale na przyszłość możesz rozważyć nazywanie propsów przekazywanych do styled components dolarem na początku np $filmImage, tak aby jednoznacznie pokazać w JSX że ten prop jest potrzebny dla stylów, a nie jest atrybutem HTML samym w sobie. Wiadomo, filmImage nie istnieje na divach, ale np jak będziesz miał disabled lub inny, to moze byc mylące. Dolar rozwiązuje sprawę.